src/app/shared/modals/specific-search/specific-search.component.scss

```diff
@@ -0,0 +1,34 @@
+@use '/src/variables.scss' as *;
+@use '@angular/material' as mat;
+
+.info {
+  color: mat.get-color-from-palette($sormas-accent, 500);
+  font-size: 12px;
+}
+
+.mat-icon {
+  font-size: 18px;
+  vertical-align: middle;
+}
+
+.delete {
+  float: left;
+  position: absolute;
+  right: 25%;
```

Comment:
this doesn't seem ok.. positions the buttons absolute

Reply:
Changed.